src/appkit.cs

```diff
@@ -23671,4 +23676,10 @@ interface NSOpenGLLayer
 		void Draw (NSOpenGLContext context, NSOpenGLPixelFormat pixelFormat, double t, ref CVTimeStamp ts);
 	}
 
+	[Protocol (IsInformal=true)]
```

Comment:
@chamons I dislike informal protocols but it's something you should *adopt* so `[Protocol]` is better IMHO (and Apple sometimes *formalize* protocols later).

In contrast a `[Category]` is something we call (and AFAIK not something we should allow overriding). 

@rolfbjarne thoughts ? ^